facebook/src/com/facebook/GraphRequest.java

```diff
@@ -606,11 +606,15 @@ public static GraphRequest newCustomAudienceThirdPartyIdRequest(AccessToken acce
         if (accessToken == null) {
             // Only use the attributionID if we don't have an access token.  If we do, then the user
             // token will be used to identify the user, and is more reliable than the attributionID.
-            String udid = attributionIdentifiers.getAttributionId() != null
-                    ? attributionIdentifiers.getAttributionId()
-                    : attributionIdentifiers.getAndroidAdvertiserId();
-            if (attributionIdentifiers.getAttributionId() != null) {
-                parameters.putString("udid", udid);
+            try {
+                String udid = attributionIdentifiers.getAttributionId() != null
+                        ? attributionIdentifiers.getAttributionId()
+                        : attributionIdentifiers.getAndroidAdvertiserId();
+                if (attributionIdentifiers.getAttributionId() != null) {
+                    parameters.putString("udid", udid);
+                }
+            } catch (NullPointerException e) {
```

Comment:
Please check that attributionIdentifiers is not null rather than catching NPE. Also attributionIdentifiers is used on 625 as well.